index.js

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-const hook = require('skygear-chat/hook');
+const skygearChatCloud = require('skygear-chat/cloud');
 const skygearCloud = require('skygear/cloud');
 const container = new skygearCloud.CloudCodeContainer();
+
 container.apiKey = skygearCloud.settings.apiKey;
 container.endPoint = skygearCloud.settings.skygearEndpoint + '/';
 
-hook.afterMessageSent((message, conversation, participants, context) => {
+skygearChatCloud.afterMessageSent((message, conversation, participants, context) => {
   const title = conversation.title;
-  const participantIds = participants.map((p) => p._id && p._id != context.userId);
+  const otherUserIds = participants.map((p) => p._id && p._id != context.userId);
```

Comment:
Per discussion, we should not change the `context.userId` in the chat sdk unless to also change the core sdk. User will be confuse we use `context.userId` in chat but `context.user_id` in core.

Btw, for sending notification, it seems using user_id in the message will be more accurate?